requirements.txt

```diff
@@ -10,6 +12,9 @@ python-dateutil==2.8.1
 pytz==2019.3
 requests==2.23.0
 six==1.14.0
+soupsieve==2.0
 tqdm==4.45.0
-urllib3>=1.25.8
+urllib3==1.25.8
 xlrd==1.2.0
+urllib3>=1.25.8
```

Comment:
The error is telling you that you have duplicate dependency

Reply:
Thanks for pointing me in the right direction, @chaosAD. Checks all pass now :)